supplementary_style_guide/glossary_terms_conventions/general_conventions/m.adoc

```diff
@@ -431,6 +441,17 @@ Azure CLI 2.0 is the most current command-line interface and is replacing Xplat-
 
 *See also*: xref:windows-server[Windows Server]
 
+[[migrate]]
+==== image:images/caution.png[with caution] migrate (verb)
+*Description*:  To _migrate_ data is to move data automatically from one location to another, usually to proactively or reactively improve cluster performance. OpenShift Container Platform uses the phrases _auto-migrate_ or _automatically migrate_ in this use case.
```

Comment:
"Migration" can refer to a lot of things and concepts, so the first sentence needs to be updated to specify the product or situation it applies to.